apps/valkyrie/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
not necessary but you can probably kill this license file since it's included in the top-level of the umbrella.

Reply:
done